### ISSUE TITLE
Enable users to create the project config via a graphical interface

### ISSUE THREAD

**Issue by niksirbi** [SainsburyWellcomeCentre/WAZP#43]
Currently, the user(s) must upload a `yaml` file with the project configuration parameters in the "Home" tab of the web app.
The idea is that users make that file once (at the beginning of each project).
As a potential enhancement, we could also provide a GUI in the home tab that allows users to create and populate the project `config.yaml` file by clicking buttons and filling input fields (like filling a form). Not a priority right now, and might be also unnecessary, if the users are comfortable with directly editing `yaml` files (we should ask for their input before we decide to implement this feature.

**Comment by sfmig** [comment]
this is a bit DLC's approach too. 

I like the simplicity of the yaml file but we can ask the researchers for their views yeah.

**Comment by sfmig** [comment]
going back to @samcunliffe's initial thought: _"Somewhere or somehow we should document that one simply needs to add a row here to define a new RoI."_ -- should we start drafting a readme for the tool? Maybe for now just assuming the project config yaml is just a yaml that the user writes

**Comment by niksirbi** [comment]
That depends on when we are expected to hand over the dashboard to users. I'd prefer proper docs over a README, and there is already an [issue](https://github.com/SainsburyWellcomeCentre/WAZP/issues/25) for that.

**Comment by sfmig** [comment]
but docs and README are non exclusive right? imo they serve different functions. Most projects typically have both, with the README giving a quick overview on installation bits and on getting started.

My suggestion is that the README could already have details on the minimum requirements for the config file, but happy to discuss if others think that is too detailed for a Readme

**Comment by niksirbi** [comment]
> but docs and README are non exclusive right? imo they serve different functions. Most projects typically have both, with the README giving a quick overview on installation bits and on getting started.

That's right, they are not mutually exclusive. But they can also overlap, e.g. we can render the contents of the `README.md` in the documentation homepage.

After my discussion with Adam and Sam today, we decide to prioritise making the docs, so I will work on that today. I will set up the necessary sphinx infrastructure, and populate the `README.md` with some basic info regarding installation, getting started, etc.
